File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PasportFileFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.passport.PassportFile;

public class PasportFileFilterValidator extends FilterValidator<PassportFile> {

    public PasportFileFilterValidator(CustomValidator<PassportFile> validator) {
        super(PassportFile.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PasportFileFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.passport.PassportFile
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class PasportFileFilterValidator(validator: Validator<PassportFile>) : FilterValidator<PassportFile>(
    PassportFile::class, validator)
